code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1456,3 +1456,14 @@ var/list/robot_verbs_default = list(
 			disable_component("comms", 160)
 		if(2)
 			disable_component("comms", 60)
+/mob/living/silicon/robot/rejuvenate()
+	..()
+	var/brute = 1000
+	var/burn = 1000
+	var/list/datum/robot_component/borked_parts = get_damaged_components(brute,burn,1)
+	for(var/datum/robot_component/borked_part in borked_parts)
+		brute = borked_part.brute_damage
+		burn = borked_part.electronics_damage
+		borked_part.installed = 1
+		borked_part.wrapped = new borked_part.external_type
+		borked_part.heal_damage(brute,burn)

```

Comment:
This should also call `borked_part.install()` - although right now, no components take an action when installed, they will soon, once my crazy mob refactor is fully processed through - most relevantly, paralyzing the borg immediately upon disabling/enabling the actuator. [`install` is called whenever a component is inserted by traditional cyborg repair.](https://github.com/ParadiseSS13/Paradise/blob/79be3ca8cfd06a56739d7b36cc095283d64fc345/code/modules/mob/living/silicon/robot/robot.dm#L590)